osbuild/main_cli.py

```diff
@@ -86,13 +87,25 @@ def parse_arguments(sys_argv):
                         help="File descriptor to be used for the monitor")
     parser.add_argument("--stage-timeout", type=int, default=None,
                         help="set the maximal time (in seconds) each stage is allowed to run")
+    parser.add_argument("--version", action="store_true",
+                        help="return the version of osbuild")
```

Comment:
There is `parser.add_argument('--version', action='version', version='<the version>')`. The docs give the following example:
```
>>> import argparse
>>> parser = argparse.ArgumentParser(prog='PROG')
>>> parser.add_argument('--version', action='version', version='%(prog)s 2.0')
>>> parser.parse_args(['--version'])
PROG 2.0
```

Reply:
I totally missed that existing, that solves the issue with `MANIFEST` as well.